tests/unit/ReviewConfirmDissolution.spec.ts

```diff
@@ -58,22 +58,6 @@ for (const test of reviewConfirmTestCases) {
       expect(wrapper.find('h2').text()).toBe('Review and Confirm')
     })
 
-    it('displays Effective Date Time section only for corp and staff', () => {
-      wrapper = shallowWrapperFactory(
-        ReviewConfirmDissolution,
-        null,
-        {
-          entityType: test.entityType,
-          tombstone: { authRoles: test.isStaff ? ['staff'] : [] }
-        },
-        null,
-        DissolutionResources
-      )
-
-      const expected = (test.entityType !== 'CP')
-      expect(wrapper.find('#effective-date-time').exists()).toBe(expected)
```

Comment:
Oh no, don't remove the unit test. It's still valuable for checking the presence of the component depending on the entity type. Just remove the bit that sets the authRoles depending on whether the user is staff or not, since the behaviour is the same in both cases.

Reply:
Ah i see, i was caught by the decoys. 
I thought the above test was for the EffectiveDateTime as well, but it's testing for the View as a whole. 
Restored the test.